core/src/main/java/com/radixdlt/api/system/system-api-schema.yaml

```diff
@@ -197,10 +197,10 @@ components:
       type: object
       required:
         - status
-        - current_fork_name
-        - executed_forks
-        - fork_vote_status
-        - unknown_reported_forks
+        - current_protocol_update_name
```

Comment:
nitpicking but maybe we should separate a "protocol update" (an action) from "protocol version"  :thinking: so this would become `current_protocol_version` or something similar.
Alternatively (if we're sticking with "protocol updates" only), `latest_protocol_update` seems to make more sense (as in "the latest protocol update that has been applied")

Reply:
I applied the final conclusions from the https://rdxworks.slack.com/archives/C01M90Y2448/p1694627607149599?thread_ts=1694625189.713549&cid=C01M90Y2448 PTAL 👍